policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml

```diff
@@ -0,0 +1,28 @@
+description: Arkansas accounts for these exemptions when calculating the state adjusted gross income.
+values:
+  2021-01-01:
+    - military_retirement_pay # Item 1
+    # Item 2&3 Life insurance policy
+    - child_support_received # Item 4
+    # Item 5 Gifts, inheritances, bequests, or devises
+    # Item 6 Scholarships, fellowships, and grants 
+    - us_govt_interest # Item 7
+    - workers_compensation # Item 8
+    - social_security # Item 8
+    # Item 9 housing allowance paid to church
+    - disability_benefits # Item 10
```

Comment:
what does the "may" relate to?

Reply:
> what does the "may" relate to?

Not quite understand.

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/481f6e67-9e5d-4b35-936c-12dd4fb0da67)

Reply:
> ![image](https://user-images.githubusercontent.com/110687043/269091697-481f6e67-9e5d-4b35-936c-12dd4fb0da67.png)

https://answerconnect.cch.com/federal/arp1209013e2c83dc42ad/irc/current/compensation-for-injuries-or-sickness

There might be some restrictions or specified circumstances when applying the disability exemption? Not sure about how to code that

Reply:
if it is a federal dependency then we can just add a comment in the list saying that